packages/flutter_tools/lib/src/features.dart

```diff
@@ -70,6 +70,11 @@ const List<Feature> allFeatures = <Feature>[
   flutterWebWasm,
 ];
 
+/// All current Flutter feature flags that can be configured.
+/// 
+/// [Feature.configSetting] is not `null`.
+Iterable<Feature> get allConfigurableFeatures => allFeatures.where((Feature feature) => feature.configSetting != null);
```

Comment:
@christopherfujino – seems there are a number of tests that assume every feature is configurable. Figured this was a clean way to address it.